cypress/utils/index.ts

```diff
@@ -2,4 +2,6 @@ export const getTestSelector = (selectorId: string) => `[data-testid=${selectorI
 
 export const getTestSelectorStartsWith = (selectorId: string) => `[data-testid^=${selectorId}]`
 
+// TODO: Remove this comment if you use the util.
+// eslint-disable-next-line import/no-unused-modules
```

Comment:
What does this mean in practice?

Reply:
I could remove it, but I just felt like it was a useful enough util to leave in the codebase despite the linter complaining.

Reply:
i vote to just remove it until it's useful